dashboards.go

```diff
@@ -0,0 +1,133 @@
+package mackerel
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+/*
+{
+  "dashboards": [
+	{
+	  "id": 2c5bLca8d,
+	  "title": "My Dashboard",
+	  "body": "# A test dashboard",
+	  "urlPath": "2u4PP3TJqbu",
+	  "createdAt": 1439346145003,
+	  "updatedAt": 1439346145003
```

Comment:
It might be better for the `createdAt` and `updatedAt` values not be the exact same, for testing purposes, maybe?